troubleshooting/known_issues/cray_console_node_in_CLBO_state.md

```diff
@@ -0,0 +1,85 @@
+# `CRAY-CONSOLE-NODE` pods intermittently goes into CLBO state after fresh CSM install
+
+## Issue Description
+
+After installing CSM 1.6, `CRAY-CONSOLE-NODE` pods intermittently goes into CLBO state. This happens because of
+permission issue with /var/log/conman directory inside the pods/container.
```

Comment:
```suggestion
After installing CSM 1.6, `cray-console-node` Kubernetes pods intermittently go into a `CrashLoopBackOff` state.
This happens because of a permission issue with the `/var/log/conman` directory inside the container.
```